go/vt/worker/row_aggregator.go

```diff
@@ -249,7 +249,7 @@ func (b *UpdatesQueryBuilder) WriteRow(buffer *bytes.Buffer, row []sqltypes.Valu
 		if i > 0 {
 			buffer.WriteByte(',')
 		}
-		sqlescape.WriteEscapeID(buffer, column)
+		sqlescape.WriteEscapeIDBytes(buffer, column)
```

Comment:
These were a bit too deep in the callstack for me to safely and confidently convert all things to strings.Builder, but I'm pretty sure it can be fine, just wanted to not touch it for right now since it's much more invasive, so opted for a version that works with `*bytes.Buffer` still.